lib/upgrade_lts_contract.py

```diff
@@ -35,13 +35,19 @@
     "18.04": "bionic",
     "20.04": "focal",
     "20.10": "groovy",
+    "21.04": "hirsute",
```

Comment:
I am adding it just for consistency sake, since we will not release the new version of the packages on those releases. But I can remove them if needed

Reply:
Regarding the test, We could start relying on `distro-info` to gather some of that info for us, but I don't think it shows what will be the next release to be installed easily. But we can perform some hacks that allow us to do that